test/unit/test-amp-inabox.js

```diff
@@ -38,6 +39,8 @@ describe('amp-inabox', () => {
       installAmpdocServices(ampdoc);
 
       const installedServicesByRegularAmp = installedServices.slice(0);
+      // The inabox mode does not need the loading indicator.
+      removeItem(installedServicesByRegularAmp, 'loadingIndicator');
```

Comment:
This seems fishy to me. I don't have the background here, but do you know why we are adding all the services by regular amp and then removing vs. only installing the correct ones to begin with?

Reply:
We not removing services. This test first collects all regular AMP services and then inabox services and then compares them all. There should be some meaningful reason why one environment should miss some of the services. But loading indicator is a legitimate case here. So I'm basically testing that `ampSevices - loadingIndicator == inaboxServices`.